### ISSUE TITLE
compliance: add CI job to verify copyright headers

### ISSUE THREAD

**Issue by mena-bosch** [eclipse/vorto#2278]
I have come across a number of resources without copyright headers, and it can be error-prone to modify the year when working on the code. 

Ideally we would have a job running on CI that verifies the following for each resource in a change set:

- Standard header present at beginning of resource
- Copyright years contain current year (so either `Copyright (c) 2020` or `Copyright (c) 2018, 2020` etc.)

Recommend @t-gauss for this one since he probably has both more knowledge of and access to our CI infrastructure.

**Comment by mena-bosch** [comment]
Also: do we want headers for front-end resources such as AngularJS files? Currently that's not in place.

**Comment by kolotu** [comment]
IMO it would be even better to use the [Maven License Plugin](https://github.com/mycila/license-maven-plugin#detailed-maven-documentation). This plugin can fail your local build, in case the license headers are not correct. 

I've configured it for all *.java files that are not generated and I have added the headers for all files that didn't have them before. 

PR is [here](https://github.com/eclipse/vorto/pull/2285)
